src/pages/{StoreCollection.slug}.tsx

```diff
@@ -76,9 +43,6 @@ function Page(props: Props) {
     [allStoreProduct]
   )
 
-  const haveYouMightAlsoLikeProducts =
```

Comment:
Do you think we can also remove the `youMightAlsoLikeProducts = useMemo` on the lines above? Saw we did that on the PDP!